source/user-manual/capabilities/vulnerability-detection.rst

```diff
@@ -59,11 +72,11 @@ The following example shows how to configure the necessary components to run the
 
   .. code-block:: xml
 
-      <wodle name="syscollector">
-        <disabled>no</disabled>
-        <interval>1h</interval>
-        <packages>yes</packages>
-      </wodle>
+    <wodle name="syscollector">
+      <disabled>no</disabled>
+      <interval>1h</interval>
+      <packages>yes</packages>
+    </wodle>
 
  Remember to restart the agent to apply changes: ``/var/ossec/bin/ossec-control restart``
```

Comment:
Maybe we should enforce using the systemctl / service command instead OSSEC binaries?